nextstrain/cli/runner/aws_batch/s3.py

```diff
@@ -187,14 +188,15 @@ def bucket(name: str) -> S3Bucket:
     """
     Load an **existing** bucket from S3.
     """
-    bucket = boto3.resource("s3").Bucket(name)
+    config = Config(retries={'max_attempts': 3})
+    s3_resource = boto3.resource("s3", config=config)
 
     try:
-        boto3.client("s3").head_bucket(Bucket = bucket.name)
+        s3_resource.meta.client.head_bucket(Bucket=name)
```

Comment:
This is an [official way](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/clients.html#creating-clients) of getting a client instance.